modules/psap-special-resource-operator-metrics.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/psap-special-resource-operator.adoc
+
+:_content-type: REFERENCE
+[id="special-resource-operator-metrics_{context}"]
```

Comment:
Ideally, your ID statement should match your filename.

Reply:
Agree, but all the files in this assembly are called "psap-" and then the real id, so I've followed that convention to keep things aligned.